src/styles/globals.ts

```diff
@@ -0,0 +1 @@
+export const className = {}
```

Comment:
I think we better reuse styles in Tailwind way:
https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply